Submitted By: Tushar Teredesai <tushar@linuxfromscratch.org>
Date: 2003-09-21
Initial Package Version: 1.1rc4
Origin: Mandarke, Redhat and Ximian RPM
Description: Prevent compilation and installation of mozilla addressbook.
OpenOffice includes precompiled mozilla libs that are linked to an older
version of gcc libs. This causes compilation failure. This patch is part
of the BLFS book.
diff -ur oo_1.1rc4_src.orig/connectivity/prj/build.lst oo_1.1rc4_src/connectivity/prj/build.lst
--- oo_1.1rc4_src.orig/connectivity/prj/build.lst	2003-06-02 02:53:47.000000000 -0500
+++ oo_1.1rc4_src/connectivity/prj/build.lst	2003-09-21 01:54:20.000000000 -0500
@@ -7,8 +7,6 @@
 cn	connectivity\source\resource			nmake	-	all	cn_res NULL
 cn	connectivity\source\sdbcx				nmake	-	all	cn_sdbcx cn_types NULL
 cn	connectivity\source\drivers\ado			nmake	-	w	cn_ado cn_dbtools NULL
-cn	connectivity\source\drivers\mozab\mozillasrc		nmake	-	all	cn_mozab_mozillasrc cn_file NULL
-cn	connectivity\source\drivers\mozab	nmake	-	all	cn_mozab cn_mozab_mozillasrc cn_dbtools NULL
 cn	connectivity\source\drivers\evoab	nmake	-	all	cn_evoab cn_dbtools cn_file NULL
 cn	connectivity\source\drivers\calc		nmake	-	all	cn_calc cn_file NULL
 cn	connectivity\source\drivers\odbc		nmake	-	all	cn_odbc cn_dbtools NULL
diff -ur oo_1.1rc4_src.orig/scp/source/office/files.scp oo_1.1rc4_src/scp/source/office/files.scp
--- oo_1.1rc4_src.orig/scp/source/office/files.scp	2003-09-05 09:57:14.000000000 -0500
+++ oo_1.1rc4_src/scp/source/office/files.scp	2003-09-21 01:55:23.000000000 -0500
@@ -388,19 +388,6 @@
 	Styles			= (PACKED);
 End
 
-#ifndef MACOSX
-File GID_FILE_LIB_MOZABDRV
-	TXT_FILE_BODY;
-	#ifdef UNX
-    Name            = STRING(CONCAT2(libmozabdrv2,UNXSUFFIX));
-	#else
-    Name            = "mozabdrv2.dll";
-	#endif
-    Dir             = GID_DIR_PROGRAM;
-	Styles			= (PACKED);
-End
-#endif
-
 STD_UNO_LIB_FILE( GID_FILE_LIB_DBU, dbu )
 STD_UNO_LIB_FILE( GID_FILE_LIB_DBA, dba)
 
@@ -750,20 +737,6 @@
         #endif
 End
 
-#ifndef MACOSX
-File GID_FILE_LIB_MOZAB_2
-        TXT_FILE_BODY;
-        Styles          = (PACKED,UNO_COMPONENT);
-        RegistryID      = GID_STARREGISTRY_SERVICES_RDB;
-        Dir             = GID_DIR_PROGRAM;
-        #ifdef UNX
-        Name            = STRING(CONCAT2(libmozab2,UNXSUFFIX));
-        #else
-        Name            = "mozab2.dll";
-        #endif
-End
-#endif
-
 #ifdef UNX
 
 File GID_FILE_LIB_EVOAB_1
